source/contour_httpproxy_test.go

```diff
@@ -1145,7 +1145,7 @@ func (ir fakeHTTPProxy) HTTPProxy() *projectcontour.HTTPProxy {
 		},
 		Spec: spec,
 		Status: projectcontour.HTTPProxyStatus{
-			LoadBalancer:  lb,
+			LoadBalancer: lb,
```

Comment:
Please avoid unrelated changes. Consider filing a separate PR for linting instead